test/property_test.c

```diff
@@ -129,6 +129,9 @@ static const struct {
     { "day", "night", "day=yes, night=yes" },
     { "", "", "" },
     { "day=yes", "-day", "day=no" },
+    { "day=yes", "-day", "day=yes" },
+    { "day=yes", "-day", "day=arglebargle" },
+    { "day=yes", "-day", "pot=sesquioxidizing" },
```

Comment:
Just to ensure that the likes of @mattcaswell and me don't start asking questions again in a year or two, it might be good to add a comment above `{ "day=yes", "-day", "day=no" },`, saying something like:

``` C
    /* The following four leave 'day' unspecified in the query, and will match any definition */

Reply:
Done.